test-driver.sh

```diff
@@ -0,0 +1,24 @@
+# Start ChromeDriver and make it non-blocking with ampersand
+./bin/chromedriver &
+
+# Keep track of the ChromeDrivers Process ID
+TASK_PID=$!
+
+# Wait for 10 seconds to give the ChromeDriver a chance to fail if there is an
+# issue.
+sleep 10
+
+# ps -p Checks if the process is still running. If it is it returns 0,
+# otherwise it returns 1
+ps -p $TASK_PID > /dev/null
+
+# Check if the process is still running by examining the exit code of ps -p
+TEST_RESULT=1
```

Comment:
`TEST_RESULT=1` will alter the result of `$?` on the next line. `$?` will always be zero.